.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 yarn-error.log
-repos/*
\ No newline at end of file
+repos/*
```

Comment:
You probably want to ignore `repos` instead of `repos/*`, since the latter lets you commit all of the repo sub-directories, which happened. You should remove them and update this.